log/DESIGN.md

```diff
@@ -0,0 +1,198 @@
+# Logs Bridge API
+
+OpenTelemetry Logs tracking issue at [#4696](https://github.com/open-telemetry/opentelemetry-go/issues/4696).
+
+## Abstract
+
+We propose adding a `go.opentelemetry.io/otel/log` Go module which will provide
+[Logs Bridge API](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/).
+
+## Background
+
+They key challenge is to create a well-performant API compliant with the specification.
+Performance is seen as one of the most imporatant charactristics of logging libraries in Go.
+
+## Design
+
+This proposed design aims to:
+
+- be specification compliant,
+- have similar API to Trace and Metrics API,
+- take advantage of both OpenTelemetry and `slog` experience to achieve acceptable performance.
+
+### Module structure
+
+The Go module consits of the following packages:
+
+- `go.opentelemetry.io/otel/log`
+- `go.opentelemetry.io/otel/log/embedded`
+- `go.opentelemetry.io/otel/log/noop`
+
+### LoggerProvider
+
+The [`LoggerProvider` abstraction](https://opentelemetry.io/docs/specs/otel/logs/bridge-api/#loggerprovider)
+is defined as an interface.
+
+```go
```

Comment:
It is fine to keep these Go declarations in this file for now, but when the actual .go files are created I think it would be better to delete them from here and just link to the actual code to avoid this content going stale and out of sync with the actual code.